website/static/js/profile.js

```diff
@@ -591,6 +603,11 @@ var ListViewModel = function(ContentModel, urls, modes) {
     self.hasMultiple = ko.computed(function() {
         return self.contents().length > 1;
     });
+
+    self.contentsLength = ko.computed(function() {
```

Comment:
This isn't necessary; just check `self.contents().length;` 

Avoid unnecessary computeds; they add extra...computation..when observables are updated.